sda-commons-client-jersey/src/test/java/org/sdase/commons/client/jersey/error/ClientRequestExceptionMapperTest.java

```diff
@@ -107,7 +110,7 @@ void mapConnectionTimeout() {
         .extracting(ApiError::getTitle)
         .asString()
         .isNotBlank()
-        .containsIgnoringCase("connect")
+        .containsIgnoringCase("Read")
```

Comment:
isn't that a semantic difference? before it was a "connect timeout" and now a "read timeout"?

Reply:
That was a good catch. Now, I had to modify the mapper because the New ConnectTimeoutException from apache 5 extends SocketTimeoutException, so it was going into the first condition. Now it is working as before:
https://github.com/SDA-SE/sda-dropwizard-commons/pull/2906/files#diff-783a06f362eb3537d9e6426d4142f1fbbcb48c683935aba914bca42ed9bd2a29R46